src/scripts/components/UserNav.js

```diff
@@ -162,17 +162,20 @@ const UserSubNavListItemLink = styled(Link)`
 class UserNav extends Component<Props, Object> {
   UserNavWrapper: HTMLElement
 
+  constructor (props: Props) {
+    super(props)
+
+    props.loadBalances()
+  }
```

Comment:
If we want to refresh the balances more, we should intentionally do it on some interval. We cannot simply do it on each `render`